README.md

```diff
@@ -1 +1,69 @@
-# Leaderboard
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To Do List
+
+> The leaderboard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to an [external API](https://www.notion.so/Leaderboard-API-service-24c0c3c116974ac49488d4eb0267ade3).
+
+>Projects Screenshots
+
+### 
+![screenshot]()
```

Comment:
- [ OPTIONAL] Kindly add a screenshot to your README.md file to make it more descriptive.